conf/modules/ins_vectornav.xml

```diff
@@ -48,4 +48,10 @@
       endif
     </raw>
   </makefile>
+  
+  <!-- For some reason this doesn't work ?
```

Comment:
there is not a good way around this - eventually NPS and HITL will be different targets, but for the time being lets leave it as is. The only INS you can use in HITL is Vectornav so doesn't really matter right now.